pkg/interop/native/crypto/crypto.go

```diff
@@ -92,3 +92,8 @@ func Bls12381Mul(x Bls12381Point, mul []byte, neg bool) Bls12381Point {
 func Bls12381Pairing(g1, g2 Bls12381Point) Bls12381Point {
 	return neogointernal.CallWithToken(Hash, "bls12381Pairing", int(contract.NoneFlag), g1, g2).(Bls12381Point)
 }
+
+// Keccak256 calls `keccak256` method of native CryptoLib contract and computes Keccak256 hash of b.
+func Keccak256(b []byte) interop.Hash256 {
```

Comment:
Need a test for it on the compiler side.

Reply:
it's in pkg/compiler/native_test.go ?

Reply:
i've added https://github.com/nspcc-dev/neo-go/pull/3301/commits/826162028a6523d920369209397a67ed126b1def#diff-38d3fe2c3c1c94577ac011836c3090e80ebe91c5fbc56aae38733a3b09a86626R242 but it has undefined: crypto.Keccak256 probably because of keccak256 not in the master

Reply:
Yes, add this test and after that create a separate commit with interop dependency update. Use `./scripts/update-deps.sh` to update the dependencies. See the example of such update: https://github.com/nspcc-dev/neo-go/pull/3155/commits/c6d259b28577725a971987157c0e0be37dd763da

After update test should pass.